src/views/admin/PersonImportView.vue

```diff
@@ -235,6 +235,7 @@
 
   function handleConfirmUnsavedChanges(): void {
     blockedNext();
+    importStore.errorCode = '';
```

Comment:
warum das?

Reply:
Wenn man einen Fehler in der Anlage hat und dann wegnavigiert über unsaved chanegs dialog und dann wieder zurück zur Anlage ist der Fehler immer noch da gewesen. Das habe ich bei allen unsaved changes dialogs korrigiert.

Reply:
Sehe ich auch so, der onBeforeRouteLeave blockt aber nur und gibt zum unsaved changes dialog weiter. Wir könnten jetzt entweder das so lassen und später refactorn oder dies wieder rückgängig machen und die Benutzer müssen dann im Zweifel die Seite neu laden, wenn es einen Fehler gibt und sie navigieren. Wäre für ersteres, weil nicht mehr so viel Zeit ist.